### PR TITLE
istioctl: point the istio support release docs for k8s support warning

### PR DESCRIPTION
**Pull request by my-git9**
**Please provide a description of this PR:**

point the istio support release docs for k8s support warning

### CODE REVIEW

File: istioctl/pkg/install/k8sversion/version.go [hunk 1]
```diff
@@ -31,7 +32,7 @@ const (
 	MinK8SVersion               = 26
 	UnSupportedK8SVersionLogMsg = "\nThe Kubernetes version %s is not supported by Istio %s. The minimum supported Kubernetes version is 1.%d.\n" +
 		"Proceeding with the installation, but you might experience problems. " +
-		"See https://istio.io/latest/docs/setup/platform-setup/ for a list of supported versions.\n"
+		"See https://istio.io/%s/docs/setup/platform-setup/ for a list of supported versions.\n"
```
Comment (hanxiaop) [inline istioctl/pkg/install/k8sversion/version.go:35]: IMO pointing to the latest is fine, the old docs are archieved, which should not be used for reference.

**Review by hanxiaop — COMMENTED** [review]
For me I would just reference https://istio.io/latest/docs/releases/supported-releases/#support-status-of-istio-releases

**Comment by my-git9** [comment]
> For me I would just refer to https://istio.io/latest/docs/releases/supported-releases/#support-status-of-istio-releases

That's great, I think it would be better if we had this link as an indicator here, what do you think?

**Comment by hanxiaop** [comment]
> > For me I would just refer to https://istio.io/latest/docs/releases/supported-releases/#support-status-of-istio-releases
> 
> That's great, I think it would be better if we had this link as an indicator here, what do you think?

Makes sense to me.

**Review by hanxiaop — APPROVED** [review]
Thanks

**Pull request merged**